src/dataAssimilation/ParameterReader.cpp

```diff
@@ -0,0 +1,29 @@
+/// \file      ParameterReader.cpp
+/// \brief     read file with only field changes.
+/// \date      Jan 03, 2021
+/// \author    My Linh Wuerzburger
+/// \copyright <2015-2021> Forschungszentrum Juelich. All rights reserved
+
+#include "ParameterReader.h"
+
+ParameterReader::ParameterReader() {
+    m_logger = Utility::create_logger(typeid(this).name());
```

Comment:
this line can be a member initializer lists

Reply:
480d503